### PR TITLE
docs(faq): add a section on text decoration

### PR DESCRIPTION
**Pull request by icy-comet**
### Prerequisites

- [x] I have read and understood the `CONTRIBUTING` guide
- [x] The commit message follows the [conventional commits][cc] guidelines
- [ ] Tests for the changes have been added (for bug fixes/features)
- [ ] Docs have been added / updated (for bug fixes/features)

### Description
- Added a section on text decoration styles not working for some terminals.
- Updated locations for PowerShell profiles according to respective versions.

[cc]: https://www.conventionalcommits.org/en/v1.0.0/#summary

### CODE REVIEW

File: docs/docs/faq.mdx [hunk 1]
```diff
@@ -157,7 +157,8 @@ Note: Tilde (~) in paths refers to your user's home directory.
 <TabItem value="powershell">
 
 ```powershell
-# ~/Documents/WindowsPowerShell/profile.ps1
+# ~\Documents\WindowsPowerShell\Microsoft.PowerShell_profile.ps1 (PowerShell v5.1 & below)
```
Comment (JanDeDobbeleer) [inline docs/docs/faq.mdx]: It's actually smarter to reference `$profile`

**Comment by JanDeDobbeleer** [comment]
@icy-comet 1 comment

File: docs/docs/faq.mdx (hunk 1 above)
Comment (icy-comet) [reply docs/docs/faq.mdx:160]: yup. ok.

Comment (icy-comet) [reply docs/docs/faq.mdx]: pushed a new commit. @JanDeDobbeleer

**Comment by icy-comet** [comment]
@JanDeDobbeleer I am so sorry. I missed the link to the powershell docs. Should I open a new PR? Or is there a way to edit the commit on your end?

Here's the missed link: https://docs.microsoft.com/en-us/powershell/module/microsoft.powershell.core/about/about_ansi_terminals?view=powershell-7.2

**Comment by JanDeDobbeleer** [comment]
@icy-comet just do a new PR

**Pull request merged**